src/clr-angular/layout/tabs/tabs.spec.ts

```diff
@@ -132,6 +132,10 @@ describe('Tabs', () => {
       context.fixture.destroy();
     });
 
+    it('sets the tab role on buttons', () => {
+      expect(compiled.querySelector('button.nav-link').getAttribute('role')).toEqual('tab');
```

Comment:
You're only checking the first `button.nav-link` here, based on the description I don't think that's what you wanted to do.

I'm guessing you want to test that the overflow toggle has the tab role, in which case you have the wrong selector.

Reply:
Actually no, this is checking that the buttons get the correct role, not the overflow button. I've added another test to check the overflow button's role.